### ISSUE TITLE
mkGraalBin.nix  { graalvm ? graalvmCEPackages.graalvm19-ce }

### ISSUE THREAD

**Issue by PlumpMath** [jlesquembre/clj-nix#94]
Hi, 
I assume, but not certain, that graalvm1x-ce has been changed to graalvm-ce, based on my speculation

Changing _graalvmCEPackages.graalvm19-ce_ to **pkgs.graalvm-ce** made the build succeed.
Thanks.

```
       error: attribute 'graalvm19-ce' missing

       at /nix/store/2k5m9diym5hq03mwc6siljd4han6qvpg-source/pkgs/mkGraalBin.nix:16:13:

           15| , version ? cljDrv.version
           16| , graalvm ? graalvmCEPackages.graalvm19-ce
             |             ^
           17|
       Did you mean graalvm-ce?
```

**Comment by jlesquembre** [comment]
Thanks for reporting. You are right, `graalvm19-ce` has been renamed to `graalvm-ce`,I'll update it. 

For more info about the change see 
https://github.com/NixOS/nixpkgs/pull/257433

**Comment by thenonameguy** [comment]
This has been fixed with https://github.com/jlesquembre/clj-nix/pull/99

**Comment by jlesquembre** [comment]
Yes, it should be fixed with #99 :)
